src/features/import/utils/prepareImportOperation.ts

```diff
@@ -0,0 +1,63 @@
+import { CellData, ColumnKind, Sheet } from './types';
+
+export type OpsType = {
+  fields?: Record<string, CellData>;
+  op: 'person.import';
+  organizations?: number | null;
+  tags?: number[];
+};
+export interface ImportOpsProp {
+  ops: OpsType[];
+}
+
+export function prepareImportOperations(configData: Sheet): ImportOpsProp {
```

Comment:
maybe call this `configuredSheet` instead of `configData`, so it's easy to remember that the type of this object is a `Sheet`?

Reply:
Yes I agree!